independent-projects/bootstrap/runner/src/main/java/io/quarkus/bootstrap/runner/RunnerClassLoader.java

```diff
@@ -1,12 +1,7 @@
 package io.quarkus.bootstrap.runner;
 
 import java.net.URL;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.Enumeration;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
please don't import `*` :)

Reply:
It seems our checkstyle check isn't picky on this: I'm reverting manually, but worth fixing it on the compilation phase IMO

Reply:
Indeed, but I think we were not able to enforce this last time we checked